src/client/app/actions/monaco.ts

```diff
@@ -39,13 +39,13 @@ export class ResetAction implements Action {
 export class UpdateIntellisenseAction implements Action {
     readonly type = MonacoActionTypes.UPDATE_INTELLISENSE;
 
-    constructor(public payload: string[], public language: string) { }
+    constructor(public payload: { libraries: string[], language: string }) {}
```

Comment:
we should be sending multiple parameters when possible. Is there a reason to this?